compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/lower_to_aie.mlir

```diff
@@ -504,6 +504,76 @@ module attributes {hal.executable.target = #executable_target_amdaie_xclbin_fb}
 
 // -----
 
+// CHECK:       aie.device(npu1_4col) {
```

Comment:
This test is for offset all zeros could you makea another  tests that has non zero offsets?

Reply:
This is also my concern.